### PR TITLE
MALAWISUP-4161

### PR DESCRIPTION
**Pull request by sradziszewski**
Added **onSupplyingFacilitySelect** function for syncs requisition supplying facility with storage

### CODE REVIEW

**Review by dariapionk — APPROVED** [review]
LGTM

File: src/requisition-convert-to-order/convert-to-order.controller.js [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+            }
+
+            $window.sessionStorage.setItem(
+                vm.selectedRequisitionsStorageKey, JSON.stringify(storageRequisitions)
+            );
+
+            setSelectAll();
+        }
+
+        // MALAWISUP-4161: Start: define new function
```
Comment (anawrotsoldevelo) [inline src/requisition-convert-to-order/convert-to-order.controller.js:277]: This comment does not say much about code it's attached to.

Comment (anawrotsoldevelo) [inline src/requisition-convert-to-order/convert-to-order.controller.js:277]: Finishing comment name should be changed too

File: src/requisition-convert-to-order/convert-to-order.controller.js [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+            loadPreviouslySelectedRequisitions();
+        };
+
+        vm.convertToOrder = convertToOrder;
+        vm.releaseWithoutOrder = releaseWithoutOrder;
+        vm.getSelected = getSelected;
+        vm.toggleSelectAll = toggleSelectAll;
+        vm.setSelectAll = setSelectAll;
+        vm.onRequisitionSelect = onRequisitionSelect;
+        // MALAWISUP-4161: Start: define new property
```
Comment (NikolaLaskowska) [inline src/requisition-convert-to-order/convert-to-order.controller.js:66]: Please change comment

File: src/requisition-convert-to-order/convert-to-order.controller.js (hunk 1 above)
Comment (sradziszewski) [reply src/requisition-convert-to-order/convert-to-order.controller.js]: Done

File: src/requisition-convert-to-order/convert-to-order.controller.js (hunk 2 above)
Comment (sradziszewski) [reply src/requisition-convert-to-order/convert-to-order.controller.js]: Done

**Review by anawrotsoldevelo — APPROVED** [review]

**Pull request merged**